wodles/utils.py

```diff
@@ -5,14 +5,18 @@
 import os
 import subprocess
 from functools import lru_cache
+from sys import exit
 
 
 @lru_cache(maxsize=None)
-def find_wazuh_path():
+def find_wazuh_path() -> str:
     """
-    Gets the path where Wazuh is installed dinamically
+    Dynamically gets the Wazuh installation path.
```

Comment:
```suggestion
    Get the Wazuh installation path.
```
Try to use infinitive for the docstrings. Review the rest of the docstrings.